MoltenVK/MoltenVK/Vulkan/vulkan.mm

```diff
@@ -2875,6 +2875,13 @@ MVK_PUBLIC_VULKAN_SYMBOL VkResult vkSetPrivateData(
 MVK_PUBLIC_VULKAN_CORE_ALIAS(vkGetDeviceMemoryOpaqueCaptureAddress, KHR);
 
 
+#pragma mark -
+#pragma mark VK_KHR_calibrated_timestamps
+
+MVK_PUBLIC_VULKAN_ALIAS(vkGetCalibratedTimestampsKHR, vkGetCalibratedTimestampsEXT);
+MVK_PUBLIC_VULKAN_ALIAS(vkGetPhysicalDeviceCalibrateableTimeDomainsKHR, vkGetPhysicalDeviceCalibrateableTimeDomainsEXT);
```

Comment:
I thought so yes, but I thought this might just be simpler for just two functions.

Reply:
It's not critical for functionality, but I agree that the `EXT` should alias from `KHR`, since promotion to `KHR` is meant to effectively deprecate the `EXT` extension, potentially (but no doubt highly unlikely) implying that the `EXT` functions might be removed at some future time. It's also consistent with how we treat all the extensions that are promoted into core.

Doing so should just involve some copy-pasting in `vulkan.mm`, so it would be best to just do it while we're thinking of it.

Reply:
yes I was going to change it but had some personal stuff and NYE going on